src/useQuery.js

```diff
@@ -7,13 +7,11 @@ const defaultOpts = {
   useCache: true
 };
 
-module.exports = function useQuery(query, opts = {}) {
+module.exports = function useQuery(query, opts = {}, updateResult) {
```

Comment:
If `update` was part of the options we don't need to update the `useQuery` signature.